### ISSUE TITLE
rasdaemon.service fails to start in Fedora 35

### ISSUE THREAD

**Issue by adatum** [mchehab/rasdaemon#57]
```
× rasdaemon.service - RAS daemon to log the RAS events
     Loaded: loaded (/usr/lib/systemd/system/rasdaemon.service; enabled; vendor preset: disabled)
     Active: failed (Result: resources)
        CPU: 0

systemd[1]: rasdaemon.service: Failed to load environment files: No such file or directory
systemd[1]: rasdaemon.service: Failed to run 'start' task: No such file or directory
systemd[1]: rasdaemon.service: Failed with result 'resources'.
systemd[1]: Failed to start RAS daemon to log the RAS events.
```

Discussion in the Bugzilla bug report https://bugzilla.redhat.com/show_bug.cgi?id=2004409 in suggests problem is an undefined  variable `SYSCONFDEFDIR` in the unit file template https://github.com/mchehab/rasdaemon/blob/master/misc/rasdaemon.service.in

**Comment by rikvanriel** [comment]
Looks like a bug in the RPM packaging.

The misc/rasdaemon.service.in file did not get processed right, leading to "EnvironmentFile=/rasdaemon" in /usr/lib/systemd/system/rasdaemon.service, and the misc/rasdaemon.env file does not appear to have been packaged at all...

Judging by misc/misc/rasdaemon.spec.in that env file was supposed to end up in /etc/sysconfig/rasdaemon, but that does not appear to have happened for some reason. Presumably the SYSCONFDEFDIR was not defined in Fedora 35?

There seems to be some code in the configure file for rasdaemon to use /etc/sysconfig as SYSCONFDEFDIR, and I'm not sure why that is failing...

**Comment by mchehab** [comment]
There were a couple of issues there. They should be fixed now, at version 0.6.8. I should be pushing it to testing later today.
